lstchain/mc/sensitivity.py

```diff
@@ -357,150 +371,159 @@ def find_best_cuts_sens(simtelfile_gammas, simtelfile_protons,
-
         eff_area[i] = e_aftercuts_w.to_value() / n_sim_bin[i] * mc_par_g['area_sim'].to(u.m**2).to_value()
 
         nevents_gamma[i] = e_aftercuts.shape[0]
         nevents_proton[i] = e_aftercuts_p.shape[0]
 
-    #Compute sensitivity  in flux units
+    # Compute sensitivity in flux units
+    egeom = np.sqrt(energy[1:] * energy[:-1])
+    dFdE, par = crab_hegra(egeom)
+    sensitivity_flux = sensitivity / 100 * (dFdE * egeom * egeom).to(u.erg / (u.cm**2 * u.s))
 
-    emed = np.sqrt(E[1:] * E[:-1])
-    dFdE, par = crab_magic(emed)
-    sens_flux = sensitivity / 100 * (dFdE * emed * emed).to(u.erg / (u.cm**2 * u.s))
-
-    list_of_tuples = list(zip(E[:E.shape[0]-2].to_value(), E[1:].to_value(), gcut, tcut,
+    list_of_tuples = list(zip(energy[:energy.shape[0]-2].to_value(), energy[1:].to_value(), gcut, tcut,
                             ngammas, nhadrons,
                             gammarate, hadronrate,
-                            nex_min, sens_flux.to_value(), eff_area,
+                            n_excesses_min, sensitivity_flux.to_value(), eff_area,
                               eff_g, eff_p, nevents_gamma, nevents_proton))
     result = pd.DataFrame(list_of_tuples,
                            columns=['ebin_low', 'ebin_up', 'gammaness_cut', 'theta2_cut',
                                     'n_gammas', 'n_hadrons',
                                     'gamma_rate', 'hadron_rate',
-                                    'nex_min', 'sensitivity','eff_area',
+                                    'n_excesses_min', 'sensitivity','eff_area',
                                     'eff_gamma', 'eff_hadron',
                                     'nevents_g', 'nevents_p'])
 
-    units = [E.unit, E.unit,"", t.unit,"", "",
+    units = [energy.unit, energy.unit,"", t.unit,"", "",
              u.min**-1, u.min**-1, "",
-             sens_flux.unit, mc_par_g['area_sim'].to(u.m**2).unit, "", "", "", ""]
-    """
-    sens_minimization_plot(eb, gb, tb, E, sens)
+             sensitivity_flux.unit, mc_par_g['area_sim'].to(u.m**2).unit, "", "", "", ""]
     
-    plot_positions_survived_events(events_g,
-                                   events_p,
-                                   gammaness_g, gammaness_p,
-                                   theta2_g, p_contained, sens, E, eb, g, t)
+    # sensitivity_minimization_plot(n_bins_energy, n_bins_gammaness, n_bins_theta2, energy, sensitivity_3Darray)
+    # plot_positions_survived_events(events_g,
+    #                               events_p,
+    #                               gammaness_g, gammaness_p,
+    #                               theta2_g, p_contained, sensitivity_3Darray, energy, n_bins_energy, g, t)
     
-    """
-    return E, sensitivity, result, units, gcut, tcut
+
+    return energy, sensitivity, result, units, gcut, tcut
 
 
-def sens(simtelfile_gammas, simtelfile_protons,
+def sensitivity(simtelfile_gammas, simtelfile_protons,
```

Comment:
"sensitivity" is also defined in "find_best_cuts_sensitivity"(sensitivity = np.ndarray(shape = n_bins_energy)).
I got the error when using this function after "find_best_cuts_sensitivity".
So it's better to change the name of one of those ones.